docs/api.md

```diff
@@ -468,7 +468,7 @@ curl http://localhost:11434/api/chat -d '{
 
 ```json
 {
-  "model": "registry.ollama.ai/library/llama3:latest",
+  "model": "ollama.com/library/llama3:latest",
```

Comment:
I know this isn't part of this change, but this isn't actually what the response model name looks like. It doesnt have the domain prefix, here is `library/llama3`
```
HTTP/1.1 200 OK
Content-Type: application/json; charset=utf-8
Date: Mon, 08 Jul 2024 18:27:33 GMT
Content-Length: 305
Connection: close

{
  "model": "llama3",
  "created_at": "2024-07-08T18:27:33.049007Z",
  "message": {
    "role": "assistant",
    "content": "My name is Mario."
  },
  "done_reason": "stop",
  "done": true,
  "total_duration": 1032541208,
  "load_duration": 788598875,
  "prompt_eval_count": 88,
  "prompt_eval_duration": 153751000,
  "eval_count": 6,
  "eval_duration": 85235000
}
```

Reply:
the model field in the response is populated by the model field in the request. it'll echo whatever is passed in:

e.g.

```
$ curl -s 127.0.0.1:11434/api/chat -d '{"model":"gemma"}' | jq -r .model
gemma
$ curl -s 127.0.0.1:11434/api/chat -d '{"model":"library/gemma"}' | jq -r .model
library/gemma
$ curl -s 127.0.0.1:11434/api/chat -d '{"model":"ollama.com/library/gemma"}' | jq -r .model
ollama.com/library/gemma
```